### DISCUSSION TITLE
main branch is locked as fork sync only

### DISCUSSION THREAD

**Discussion by cbugk** [cbugk/vuejs-docs-tr#1] [Announcements]
@exrem I have cached your commit from 2024-01-05 into branch [user-exrem](https://github.com/cbugk/vuejs-docs-tr/tree/user-exrem) temporarily. Feel free to delete the commit and use that branch as a workbench.

Also create a pull request from a seperate new branch with your changes please.

**Comment by exrem** [comment]
'kay, and if you don't mind, i've got a question cuz i'm just wondering if we gotta use github with that level of _professionality_ :D

**Reply by cbugk** [reply]
It is for ease of syncing to upstream repository. I don't think we need to go all-in at the beginning. I created `docs-tr` as a proxy main branch. Let's face the fact that it is only a two-person endeavor for now and just get things done. However, I would like to have 2 sets of eyes on every commit/translation anyway. So separate issues for folders/files with their own branch, which are PR'ed onto `docs-tr`, seems a good balance to me.

**Reply by exrem** [reply]
all right, thanks ^^

**Reply by exrem** [reply]
umm, i wonder if i'm doing this the right way so if there's any mistakes i can fix them before sending more prs @cbugk?

**Reply by cbugk** [reply]
@exrem it was a quite short file anyway, let's not undo the merge for issue #2. I added a requirement of 2 approvals for __docs-tr__ branch. That should suffice for the time being. You can create issues and assign to yourself. Just make sure to base your [development branches](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-a-branch-for-an-issue) from __docs-tr__ rather than the default one, __main__.

P.S.: It is my first time trying to set a collaborative repository structure, so we will figure as we go. Normally I would  try sticking to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for peace of mind but we are not doing [Semantic Versioning](https://semver.org/) nor any versioning for that matter. Let's stick to the job at hand for now. Keep them PRs coming on your own pace, and I will do the same.
